condition.go

```diff
@@ -117,3 +120,44 @@ func Lte(column string, value interface{}) Builder {
 		return buildCmp(d, buf, "<=", column, value)
 	})
 }
+
+func buildLikeCmp(d Dialect, buf Buffer, pred string, column string, value interface{}) error {
+	if value == nil {
+		return fmt.Errorf("Column %s %s nil is Not Supported", column, pred)
+	}
+
+	v := reflect.ValueOf(value)
+	switch {
+	case v.Kind() == reflect.TypeOf([]rune{}).Kind() && v.Len() != 0:
```

Comment:
Sorry, the empty int slice and the byte slice were caught on Detect of the Rune slice so I temporarily left it.
I will fix it with the next commit so please wait for a while.